runtime/frequency/src/lib.rs

```diff
@@ -1406,6 +1404,7 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 	type WeightInfo = ();
 	type ConsensusHook = ConsensusHook;
 	type SelectCore = DefaultCoreSelector<Runtime>;
+	type RelayParentOffset = ConstU32<0>;
```

Comment:
nit: Are we sure we want to set this to zero? It might help with forks @wilwade 

```
		/// While this helps to reduce forks on the parachain side, it increases the delay for
		/// processing XCM messages. So, the value should be chosen wisely.

```

Reply:
Read somewhere that for most cases 0 is okay but yes this is one thing we should investigate, @enddynayn  might have some ideas